common/src/main/java/com/datatorrent/common/metric/MetricsAggregator.java

```diff
@@ -129,13 +128,13 @@ public void addAggregators(@NotNull String metric, @NotNull SingleMetricAggregat
    *                           be used for the result of aggregators[i].
    */
   public void addAggregators(@NotNull String metric, @NotNull SingleMetricAggregator[] aggregators,
-                             @NotNull String[] logicalMetricNames)
+      @NotNull String[] logicalMetricNames)
   {
     Preconditions.checkNotNull(metric, "metric");
     Preconditions.checkNotNull(aggregators, "aggregators");
     Preconditions.checkNotNull(logicalMetricNames, "logicalMetricNames");
     Preconditions.checkArgument(aggregators.length == logicalMetricNames.length, "different length aggregators and" +
-      " logical names");
+        " logical names");
```

Comment:
Avoid split on the string literal. Can the previous line be split after the comma?